### ISSUE TITLE
Hotkey List: local const workflowPath in script.js

### ISSUE THREAD

**Issue by lisaross** [rknightuk/alfred-workflows#10]
Just wanted to leave a note here for anyone else - to make this work you need to modify the const workflowPath in line 1 to your local file path ie: 

`const workflowPath = '/Users/lisaross/Dropbox/Lisa Inbox/App Stuff/Alfred.alfredpreferences/workflows';`

Found in debug: [13:58:24.888] ERROR: Hotkey List[Script Filter] Code 1: ./script.js: execution error: Error: Error: Can't get object. (-1728)

Thanks @rknightuk for all your gorgeous and super useful workflows!

**Comment by rknightuk** [comment]
I thought I'd done that to a workflow variable, will get that updated when I get chance. Thanks!

**Comment by CullumAttwell** [comment]
Thanks for all your workflows rknightuk - I've been downloading a few tonight and they have been really useful. I wondered whether you ended up getting round to this? I have checked the workflow but am a bit stuck with how to add the above information.

*Edit: I figured out how to sort this out based on lisaross's comment (thank you). I thought it might be useful to document more clearly what needs altering for those, like me, who are less tech-savy.
0. Double click on the "Run Script" item.
1.  Open up the folder 
<img width="819" alt="image" src="https://github.com/rknightuk/alfred-workflows/assets/80011479/78d35264-7d25-48d5-b083-439a23f60040">
2. Open the script (you'll need an app like BBEdit to do this:
<img width="702" alt="image" src="https://github.com/rknightuk/alfred-workflows/assets/80011479/91a98420-2d0c-43ed-a9e0-071668f2f8f6">

3. Copy the file path of where this is stored up until "workflows".
4. Copy the file path into the first line: "const workflowPath = '/Users/XXX/Dropbox/Lisa Inbox/App Stuff/Alfred.alfredpreferences/workflows';", where XXX should be your User name.
<img width="868" alt="image" src="https://github.com/rknightuk/alfred-workflows/assets/80011479/96f5ffa8-49a1-41b3-8e5d-c0df6ef0cee7">
